### PR TITLE
Software testing

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**
Chapter 6 created 👍

### CODE REVIEW

**Comment by udeysingh95** [comment]
There's one line repeating 4 times. 
"This will be covered in.." 
Keep it only once. Choose the right place yourself. 
Rest is looking good.

File: README.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ Apply graph and string algorithms to solve real-world challenges: finding shorte
 
          3. Agile Methods       4. Project Planning
 
-         5. Software Design
- 
+         5. Software Design     6. Software Testing(basics)
```
Comment (udeysingh95) [inline README.md:53]:
```suggestion
         5. Software Design     6. Software Testing 
```

**Comment by Vaibhavpratapsingh22** [comment]
@udeysingh95 
I had done this change.

**Comment by udeysingh95** [comment]
Its not reflected yet.

**Pull request merged**